src/app/oss-routing.module.ts

```diff
@@ -280,18 +302,30 @@ const routes: Routes = [
       { path: "setup/families-for-enterprise", component: FamiliesForEnterpriseSetupComponent },
     ],
   },
+  {
+    path: "organizations",
+    component: OrganizationLayoutComponent,
+    canActivate: [AuthGuardService, OrganizationRedirectGuardService],
+    data: {
+      permissions: organizationRoutePermissions.all(),
+    },
+    pathMatch: "full",
+  },
```

Comment:
What do you think of moving the organization routes to it's own router similar to how we've begun splitting it up in the other PRs?

As a way to begin breaking up the application in better modules.

Reply:
I think it's a good idea, I've created a separate routing module, I'm now creating a whole separate `organizations.module.ts` as well. You gotta do the PR review though :P

Reply:
Update: I've added the routing module here, will leave the app module for a separate refactor/PR.